hdrass/CCW+Rotator_integration_testing/Rotator_interlock_test.ipynb

```diff
@@ -0,0 +1,1248 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(mtm, salobj.State.ENABLED)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Move to Zero"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Move to 0.0 deg starting position\")\n",
+    "\n",
+    "await rot.cmd_move.set_start(position=0.0,timeout=30.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(mtm, salobj.State.STANDBY)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Wait for override off"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(mtm, salobj.State.ENABLED)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Set Low Velocity and Move Through Positive Interlock\n",
+    "This will move the rotator indepently from the CCW and will therefore activate the positive limit switch located at the bulkhead plate between the Rotator and the CCW"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#await rot.cmd_configureVelocity.set_start(vlimit=1.5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Disable the CCW"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "CCWstate = mtm.evt_cameraCableWrapFollowing.get()\n",
+    "print(\"CCWstate:\",CCWstate.enabled)\n",
+    "await mtm.cmd_disableCameraCableWrapFollowing.set_start(timeout=10.)\n",
+    "await CCWstate = mtm.evt_cameraCableWrapFollowing.next(timeout=10.)\n",
+    "print(\"CCWstate:\",CCWstate.enabled)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This will make the rotator fault"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "await asyncio.sleep(1.)\n",
+    "\n",
+    "print(\"Move to +10 deg position\")\n",
+    "\n",
+    "await rot.cmd_move.set_start(position=+10,timeout=30.)\n",
+    "\n",
+    "print(\"Reset the Rotator and CCW\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Bring the Rotator and CCW back to enabled state"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "await mtm.cmd_clearerror.start(timeout=30)"
```

Comment:
Note that the command is actually `clearError` (capital E), it is not yet implemented and I kind of hope we'll never need it. (Also, as presently documented it wants a mask of what to reset).

Reply:
Oki, this was for an old version where clearError was a substantial part of any execution. Left here for documentation and will be changed for the next round of tests. Noted at the beginning of the notebook.